services/order.py

```diff
@@ -0,0 +1,38 @@
+from django.contrib.auth import get_user_model
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, MovieSession
+
+
+def create_order(
+        tickets: list[dict],
+        username: str,
+        date: str = None
+) -> None:
+    with transaction.atomic():
+        user = get_user_model().objects.get(username=username)
+        new_order = Order.objects.create(user_id=user.id)
+        if date:
+            new_order.created_at = date
+
+        new_order.save()
```

Comment:
`save()` will be calls twice (for the first time during creation of `new_order`) no matter if date provided, so it's better to move `save()` into `if` body.